mtp_api/apps/mtp_auth/views.py

```diff
@@ -200,7 +224,7 @@ def post(self, request):
 
             send_email(
                 user.email, 'mtp_auth/reset_password.txt',
-                gettext('Your new Money To Prisoners password'),
+                gettext('Your new Prisoner Money password'),
```

Comment:
Has Florian or someone officially decided that we're changing the user facing instances of the service name or is this you taking the initiative?